compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -6459,10 +6459,15 @@ private List<BLangNamedArgsExpression> checkProvidedErrorDetails(BLangErrorConst
         for (BLangNamedArgsExpression namedArgsExpression : errorConstructorExpr.namedArgs) {
             BType target = checkErrCtrTargetTypeAndSetSymbol(namedArgsExpression, expectedType);
 
+            namedArgsExpression.cloneAttempt++;
             BLangNamedArgsExpression clone = nodeCloner.cloneNode(namedArgsExpression);
```

Comment:
`cloneNode` also increments the `cloneAttempt`, right? Do we know why we need this here?